src/CMakeLists.txt

```diff
@@ -14,7 +15,6 @@ endif()
 add_subdirectory(debug)
 add_subdirectory(drivers)
 add_subdirectory(init)
-add_subdirectory(math)
```

Comment:
Why is the ordering changing here ?

Reply:
I have to change this order, because I had an error with undefined reference to crc32() function for HOST building (it was not in "libsof"). If you prefer, I can add new file for host/testbench with crc32 definition (the same as for FW) and back to "previous" order.